### ISSUE TITLE
web code

### ISSUE THREAD

**Issue by QuocLeIT** [QuocLeIT/web-code#1]
<!DOCTYPE html>
<html lang="en">
<head>
  <title>Home</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
  <link rel="icon" href="http://icons.iconarchive.com/icons/iconshock/world-of-warcraft/256/warcraft-1-icon.png">
    
  <style>

    body {
       background-image: url("http://www.ngoclopoto.com/upload/product/08919531653.jpg");
       background-repeat: no-repeat;}
    div.background-image {
    /*background-image: url("E:\Lap trinh\trang web08919531653.jpg"); tai s lam nhu the nay thi k duoc*/
    	background: url(http://www.ngoclopoto.com/upload/product/08919531653.jpg);
    	text-align: center;
    	color: #f2f2f2;
      margin: 0;
      padding: 30px 300px 80px 300px;}
    /*phan tieu de cuoi trang*/
    div.background-color {
      background-color: #1a1a1a;
      margin: 0;
      padding: 20px 30px 50px 10px;}
    /*mau cho phan Our*/
    div.background-color2 {
        background-color: #262626;
        padding: 20px 50px 20px 50px;}

    div.img {
      margin: 30px;
      float: left;
      width: 220px;}
    div.padding {
      padding: 50px 50px;
    }
    /*tao cac danh muc Home About ...*/
    ul {
    	list-style-type: none;
    	padding: 0;
      margin: 0;
    	overflow: hidden;
    	background-color: none;}
    ul.left {margin-left: 500px;}
    ul.right {margin-right: 650px;}
    /*li:hover {background-color:  #ff1a1a;}/*doi mau nen cua o: mau do*/
    li {float: left;}
    li a {
      display: block;
      color: white;
      text-align: center;
      text-decoration: none;
    }
    li.TieuDeDauTrang {padding: 14px 50px;height: 150px;}
    .TieuDeDauTrang:hover {background-color: #ff1a1a;}
    li.Trang {
      background-color: #ff1a1a;
      padding: 14px 50px;
      height: 150px;}
    li.TieuDeCuoiTrang {padding: 7px 25px;}
    .Mau {background-color: #ff1a1a;}
    /*li a:hover:not(.active) {background-color: none;}/*doi mau nen cua chu: khong mau*/
    	
    body {font-family: "Arial", Times, Sans Serif;}
    p.{font-family: "Arial", Times, Sans Serif;}
    p.TieuDe {font-size: 40px;}
   
    .center {
      position: absolute;
      font-size: 18px;
      left: 20%;
      top: 50%;
      width: 100%;
    }
    /*div.img:hover {border: 1px solid #777;}*/
    img.right {
    float: right;
    margin: 0 75px 10px 10px;}
    img.icon { border-radius: 50%;}

    div.img img {
      width: 100%;
      height: auto;}

    div.desc {
      margin: 5px;
      padding: 0px;
      text-align: left;}

    div.text {
      margin: 0px;
      padding-left: 510px;
      color: #e6e6e6;
      text-align: left;}
    div.background-OurServices {background-color: #ddd;}
    
    .paddingTruck {
       padding-left: 100px;
       padding-right: 700px;}
    .TextRed {color: red;}
    .TextBlack {color: black;}
    .Textwhite {color: #f2f2f2;}
    .TextCenter {
      text-align: center;
      padding: 60px 150px 80px 150px;
      font-size: 20px;}
    .xam {color: #8c8c8c}
    .padding-left {padding-left: 90px;}
    .padding-right {padding-right: 600px;}
    .padding-top-bottom {
      padding-top: 20px;
      padding-bottom: 20px;}

    a:link { line-height: none; background-color: none;}
    a:hover {
      color: white;
      text-decoration: none;
      background-color: none;}/*dua chuot vao link thi co mau den*/
      a.is:hover { color: black;}
    a:active {
      text-decoration: none;
      background-color: none;}
    
    input[type=text] {
      width: 40%;
      border-radius: 4px;
      font-size: 16px;
      color: #595959;
      background-color: #1a1a1a;
      background-position: 10px 10px; 
      background-repeat: no-repeat;
      padding: 12px 20px 12px 10px;
      border-bottom: 2px solid #404040;}

    input[type=button]{
      background-color: #777;
      border: none;
      color: white;
      padding: 10px 20px;
      text-decoration: none;
      margin: 4px 2px;
      cursor: pointer;}
    .btn_subscribe:hover {
    background-color: #ff1a1a;}
    .service {
      padding: 20px;
      height: 250px;
      /*background-color: #ddd;/*mau cua khung*/
      text-align: center;
      -webkit-transition-duration: 1s;
      -o-transition-duration: 1s;
      -moz-transition-duration: 1s;
      -ms-transition-duration: 1s;
      transition-duration: 1s;}
    .service:hover {background-color: #ff8080;}
    .background-SeeMore {background-color: #777;}
    .background-SeeMore:hover {background-color: #ff1a1a;}

  </style>
</head>
<body>
  <li class="padding-left"><a href="#"><img src="http://www.isource.vn/template/1436675954/images/logo.png" alt="" /></a></li>
  <ul class="left">
    <li class="Trang"><a href="http://www.isource.vn/template/1436675954/index.html"><b>Home<br><br><span class="menu"><img src="http://www.isource.vn/template/1436675954/images/nav-but1.png" alt=""></span></b></a></li>
    <li class="TieuDeDauTrang"> <a href="http://www.isource.vn/template/1436675954/about.html"><b>About<br><br><span class="menu"><img src="http://www.isource.vn/template/1436675954/images/nav-but2.png" alt=""></span></b></a></li>
    <li class="TieuDeDauTrang"> <a href="http://www.isource.vn/template/1436675954/services.html"><b>Servise<br><br><span class="menu"><img src="http://www.isource.vn/template/1436675954/images/nav-but3.png" alt=""></span></b></a></li>
    <li class="TieuDeDauTrang"> <a href="http://www.isource.vn/template/1436675954/blog.html"><b>Blog<br><br><span class="menu"><img src="http://www.isource.vn/template/1436675954/images/nav-but4.png" alt=""></span></b></a></li>
    <li class="TieuDeDauTrang"> <a href="http://www.isource.vn/template/1436675954/mail.html"><b>Mail us<br><br><span class="menu"><img src="http://www.isource.vn/template/1436675954/images/nav-but5.png" alt=""></span></b></a></li>
  </ul>

  <div class="TextCenter">
    <p class="TieuDe"><h class="Textwhite">Where you <b>always</b> find great truck journey</h></p>
    <p class="Textwhite">Ut sodales erat tortor, eget rhoncus nulla rutrum sit amet. Aliquam sit amet lorem dui. Nulla sagittis dolor id mi tincidunt varius. Donec quis suscipit tortor vel pellentesque libero</p><br><br>

    <p class="TieuDe"><h class="Textwhite">Make your <b>journey</b> truck in United Kingdom</h></p>
    <p class="Textwhite">Eget rhoncus nulla rutrum sit amet. Ut sodales erat tortor Aliquam sit amet lorem dui. Donec quis suscipit tortor vel pellentesque libero Nulla sagittis dolor id mi tincidunt varius</p><br><br>

    <p class="TieuDe"><h class="Textwhite">Provider <b>Volvo Trucks</b> truck in Ukraine</h></p>
    <p class="Textwhite">Eget rhoncus nulla rutrum sit amet. Ut sodales erat tortor Aliquam sit amet lorem dui. Donec quis suscipit tortor vel pellentesque libero Nulla sagittis dolor id mi tincidunt varius</p>
  </div>

  <img class="right" src="http://www.isource.vn/template/1436675954/images/2.jpg" alt="Northern Lights" width="450" height="250"> 

  <div class="padding-right"><div class="padding-left">
    <p class="TieuDe"> Nullam consectetur tristique fermentum vestibulum </p>
    <p class="xam"> Cras porttitor imperdiet volutpat. Nulla malesuada lectus eros ut convallis felis consectetur ut</p>
    <p>Proin eget ipsum ultrices, aliquet velit eget, tempus tortor. Phasellus non velit sit amet diam faucibus molestie. Mauris sapien eros, mattis et elit non, tincidunt efficitur nisi augue dui iaculis nulla, a pretium nisl turpis vel augue auctor viverra aliquam .</p>

    <a href="http://www.isource.vn/template/1436675954/about.html"><input class="btn_subscribe" type="button" value="SEE MORE"></a>
  </div></div>

  <br><br><br>

  <div class="background-OurServices"><div class="container">
    <div class="col-md-5">
      <p class="TieuDe"><h class="TextRed">Our Services</h></p>
      <div class="padding-top-bottom"><p class="xam">Cras porttitor imperdiet volutpat nulla malesuada lectus eros ut convallis felis consectetur ut</p></div>
      <div class="padding-top-bottom"><p> Integer vitae ligula sed lectus consectetur pellentesque blandit nec orci. Nulla ultricies nunc et lorem semper, quis accumsan dui aliquam aucibus sagittis placerat.</p></div>

      <div class="padding-top-bottom"><p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Morbi non nibh nec enim sollicitudin interdum.tristique senectus et netus et malesuada fames ac turpis egestas</p></div>

      <div class="padding-top-bottom"><a href="http://www.isource.vn/template/1436675954/services.html"><input class="btn_subscribe" type="button" value="SEE MORE"></a></div>
    </div>

    <div class="col-md-7">
      <div class="col-md-6 service">
        <figure class="icon"><img class="icon" src="http://www.isource.vn/template/1436675954/images/1.png"    /></figure>
        <p><b>Proin eget ipsum ultrices</b><br><br>Sed ut perspiciis iste natus error sit voluptatem accusantium doloremque laudantium.</p></div>

      <div class="col-md-6 service">
        <figure class="icon"><img src="http://www.isource.vn/template/1436675954/images/2.png" alt="" /></figure>
        <p><b>Proin eget ipsum ultrices</b><br><br>Sed ut perspiciis iste natus error sit voluptatem accusantium doloremque laudantium.</p></div>

      <div class="col-md-6 service">
        <figure class="icon"><img src="http://www.isource.vn/template/1436675954/images/3.png" alt="" /></figure>
        <b>Proin eget ipsum ultrices</b><br><br>Sed ut perspiciis iste natus error sit voluptatem accusantium doloremque laudantium.</div>
      
      <div class="col-md-6 service">
        <figure class="icon"><img src="http://www.isource.vn/template/1436675954/images/4.png" alt="" /></figure>
        <b>Proin eget ipsum ultrices</b><br><br>Sed ut perspiciis iste natus error sit voluptatem accusantium doloremque laudantium.</div>
     </div>
  </div></div>

  <br><br><br>

  <div class="background-image">
    <p class="TieuDe">Testimonials</p>
    <p>Cras porttitor imperdiet volutpat nulla malesuada lectus erosut convallis felis consectetur ut</p>
    <i><b><p>" Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam fermentum iaculis diam quis sodales. Vestibulum eu dui tellus. In viverra porttitor auctor. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas "</p></b></i>
  </div>

  
  <br>
  <div class="padding">
  <div class="padding-left">
    <p class="TieuDe"><h class="TextRed">News</h></p>
    <p class="xam">Cras porttitor imperdiet volutpat nulla malesuada lectus eros ut convallis felis consectetur ut</p></div>

  <div class="img">
    <b><a class="TextBlack" href="http://www.isource.vn/template/1436675954/web-0.html">Integer vitae ligula sed lectus</a></b>
    <p class="TextRed">8.00 - 10.00 | JUN 09,2014</p>
    <a target="_blank" href="E:\Lap trinh\trang web\img1.jpg">
    <img src="http://www.isource.vn/template/1436675954/images/img1.jpg" alt="Trolltunga Norway" width="300" height="200"></a>
    <div class="desc"><p>Pellentesque ut urna eu mauris scele risque auctor volutpat et massa pers piciis iste natus scele risque auctor volutpat et massa.</p></div>
  </div>

  <div class="img">
    <b><a class="TextBlack" href="http://www.isource.vn/template/1436675954/web-0.html">Integer vitae ligula sed lectus</a></b>
    <p class="TextRed">10.00 - 12.00 | sep 24,2014</p>
    <a target="_blank" href="E:\Lap trinh\trang web\img2.jpg">
      <img src="http://www.isource.vn/template/1436675954/images/img2.jpg" alt="Forest" width="600" height="400"></a>
    <div class="desc"><p>Pellentesque ut urna eu mauris scele risque auctor volutpat et massa pers piciis iste natus scele risque auctor volutpat et massa.</p></div>
  </div>

  <div class="img">
    <b><a class="TextBlack" href="http://www.isource.vn/template/1436675954/web-0.html">Integer vitae ligula sed lectus</a></b>
    <p class="TextRed">9.00 - 10.00 | FEB 15,2014</p>
    <a target="_blank" href="E:\Lap trinh\trang web\img3.jpg">
      <img src="http://www.isource.vn/template/1436675954/images/img3.jpg" alt="Northern Lights" width="600" height="400"></a>
    <div class="desc"><p>Pellentesque ut urna eu mauris scele risque auctor volutpat et massa pers piciis iste natus scele risque auctor volutpat et massa.</p></div>
  </div>

  <div class="img">
    <b><a class="TextBlack" href="http://www.isource.vn/template/1436675954/web-0.html">Integer vitae ligula sed lectus</a></b>
    <p class="TextRed">11.00 - 10.00 | JUN 10,2014</p>
    <a target="_blank" href="E:\Lap trinh\trang web\img4.jpg"><img src="http://www.isource.vn/template/1436675954/images/img4.jpg" alt="Mountains" width="600" height="400"></a>
    <div class="desc"><p>Pellentesque ut urna eu mauris scele risque auctor volutpat et massa pers piciis iste natus scele risque auctor volutpat et massa.</p></div>
  </div></div>

  <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>

  <div class="background-color">
    <footer class="container"><div class="row">
      <div class="col-md-12">
        <ul class="right">
          <li class="TieuDeCuoiTrang"> <a href="http://www.isource.vn/template/1436675954/index.html">Home</a></li>
          <li class="TieuDeCuoiTrang"> <a href="http://www.isource.vn/template/1436675954/about.html">About</a></li>
          <li class="TieuDeCuoiTrang"> <a href="http://www.isource.vn/template/1436675954/services.html">Servise</a></li>
          <li class="TieuDeCuoiTrang"> <a href="http://www.isource.vn/template/1436675954/blog.html">Blog</a></li>
          <li class="TieuDeCuoiTrang"> <a href="http://www.isource.vn/template/1436675954/mail.html">Mail us</a></li>
        </ul>
        <form><input type="text" name="Email" placeholder="Email"></form><input class="btn_subscribe" type="button" value="Subscribe">
      </div>

      <div class="col-md-6">
        <h3 class="Textwhite">Address</h3>
        <p class="Textwhite">756 gt globel Place, CD-Road,M 07 435.<br>Phone: +1(100)2345-6789</p>
      </div>
      <div class="col-md-6">
        <a href="#"><img src="http://www.isource.vn/template/1436675954/images/logo.png" alt="" /></a>
         <p class="Textwhite"><br>Proin eget ipsum ultrices, aliquet velit eget, tempus tortor.<br>Phasellus non velit sit amet diam faucibus molestie tincidunt efficitur nisi.</p>
      </div>

      <ul class="pagination">
        <li><a href="#">Home</a></li>
        <li><a href="#">About</a></li>
        <li><a href="#">Services</a></li>
        <li><a href="#">Blog</a></li>
        <li><a href="#">Mail</a></li>
      </ul>
  
  </div></footer></div>

  <div class="background-color2">
    <p class="Textwhite"> <b>© 2015 Truck. Design by <a class="Textwhite" href="http://isource.vn/">isource</b></a></p>
  </div>
</body>
</html>
